### DISCUSSION TITLE
Unwanted prompt expansion

### DISCUSSION THREAD

**Discussion by Riveda72** [lllyasviel/Fooocus#3166] [Q&A] [answered]
Hi, I'm a newbie with Fooocus and Pinokio.
Is there a way to disable the prompt expansion? It adds unwanted parameters wich totally contraddict even my negative prompt, so I get results totally far from what I expected. 
Actually I also wonder what's the point in writing precise prompts and negative prompts, if the application just adds by its own initiative "expanded prompts" that contraddict the ones added by the user...

`[Prompt Expansion] {my prompt here}, intricate, elegant, highly detailed, dynamic composition, sharp focus, color, rich deep colors, ambient dramatic cinematic light, epic, spectacular, very inspirational, beautiful, stunning, creative, vibrant, thought inspired, cute, perfect, pretty, **innocent**, enhanced, coherent, complex, iconic, fine, **illuminated**, fair, amazing, marvelous, imposing, awesome, enormous`

Just consider that I asked for a nightscape ("illuminated" added by the prompt expansion) and an assassin character, which is absolutely not "innocent" as unwantedly added by the promp extension.

**Comment by ILikeToasters** [comment, marked as answer]
Just uncheck any styles if you don't want them used. Fooocus V2 is the dynamic one and the rest always add the same things depending on which style is used. 
<img width="378" alt="Screenshot 2024-06-19 200209" src="https://github.com/lllyasviel/Fooocus/assets/4357657/e73a27d0-98e5-46c4-80b4-497dd4d2ca87">

**Reply by Riveda72** [reply]
Done, thanks a lot :)
